docs/spring-boot-runtime/master.adoc

```diff
@@ -12,6 +12,12 @@ include::topics/templates/document-attributes.adoc[]
 
 This guide covers concepts as well as practical details needed by developers to use the {SpringBoot} runtime. It provides information governing the design of a {SpringBoot} application deployed as a Linux container on OpenShift.
 
+[IMPORTANT]
+--
+This guide applies to {SpringBoot} {version-spring-boot}.
+For releases based on {SpringBoot} 1.5.x, follow the link:{link-guide-spring-boot-1-5}[{SpringBoot} 1.5.x Runtime Developer Guide].
```

Comment:
Replace the link text with the `{name-guide-spring-boot-1-5}` variable reference?

Reply:
will do.